src/main/scala/no/ndla/searchapi/integration/GrepApiClient.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Part of NDLA search-api.
+ * Copyright (C) 2020 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.searchapi.integration
+
+import java.util.concurrent.Executors
+
+import com.typesafe.scalalogging.LazyLogging
+import no.ndla.network.NdlaClient
+import no.ndla.searchapi.SearchApiProperties.GrepApiUrl
+import no.ndla.searchapi.model.api.GrepException
+import no.ndla.searchapi.model.domain.RequestInfo
+import no.ndla.searchapi.model.grep._
+import org.json4s.DefaultFormats
+import scalaj.http.Http
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, ExecutionContextExecutor, Future}
+import scala.util.{Failure, Success, Try}
+
+trait GrepApiClient {
+  this: NdlaClient =>
+  val grepApiClient: GrepApiClient
+
+  class GrepApiClient extends LazyLogging {
+    implicit val formats: DefaultFormats.type = org.json4s.DefaultFormats
+    private val GrepApiEndpoint = s"$GrepApiUrl/kl06/v201906"
+
```

Comment:
Du må sannsynligvis også ha kompetansemålsett og læreplaner også. Det er ikkje heilt avgjort kva data som trengs indeksert for forskjellige artikler.

Reply:
Taksonomi har støtte for å sette grepcodes på subject, topic og resource. Dersom for eksempel koden til et kompetansemålsett er satt på et topic, kan det fort tenkes at vi må hente ut alle kompetansemål fra settet og putte det på artikkelen.

Reply:
Skal jeg da la ting stå som de er og vi kan utvide dem senere når vi vet hva vi trenger, eller skal pr holdes åpen også er det noe dere kan avklare i nærmeste dagene sånn at jeg kan implementere det i denne pr?

Reply:
Eg skal være med på et møte i morra. Håper vi får avklaringer då.

Reply:
Får ikkje avklaring på dette på kort sikt så vi kan fullføre pr.